### DISCUSSION TITLE
make mint ARGS="--network sepolia" , ERROR

### DISCUSSION THREAD

**Discussion by yasharyan0904** [Cyfrin/foundry-full-course-cu#1908] [Q&A] [answered]
Error: 
script failed: unknown selector `0xdb4235f6` for VmCalls
Reaping losing child 0x5639d9744cc0 PID 121161 
make: *** [Makefile:43: mint] Error 1
Removing child 0x5639d9744cc0 PID 121161 from chain.

**Comment by EngrPips** [comment, marked as answer]
Hello @yasharyan0904, Have you tried to run `foundryup` and then retry? Foundry sometimes behaves weirdly when it needs an update.

**Reply by yasharyan0904** [reply]
Yep, it worked :)

**Reply by EngrPips** [reply]
Awesome. Keep crushing it, friend.
